tests/artist-create.test.js

```diff
@@ -0,0 +1,27 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create artist', () => {
+  describe('/artists', () => {
```

Comment:
Feel like there is one too many describe blocks here, without any beforeEach it seems a little unnecessary imo
```
```